WordPressAuthenticator/NUX/NUXViewControllerBase.swift

```diff
@@ -71,9 +71,9 @@ extension NUXViewControllerBase where Self: UIViewController, Self: UIViewContro
 
     /// Displays a login error in an attractive dialog
     ///
-    func displayError(_ error: NSError, sourceTag: WordPressSupportSourceTag) {
+    func displayError(_ error: Error, sourceTag: WordPressSupportSourceTag) {
```

Comment:
Inspired by one of your PRs that I reviewed earlier where `authenticateWithUsername(` became `authenticate(username:`, what do you think of making this 

```suggestion
    func display(error: Error, sourceTag: WordPressSupportSourceTag) {
```

Or even simply

```suggestion
    func display(_ error: Error, sourceTag: WordPressSupportSourceTag) {
```

Reply:
👍 Sounds good! There are more changes to come. I'll leave them as they are for now, and change them in my next PRs.